backend/src/db/migrations/20230901145235_create_prompts_table.ts

```diff
@@ -0,0 +1,59 @@
+import { type Knex } from 'knex';
+
+const TableName = {
+  PROMPTS: 'prompts',
+  GENRES: 'genres',
+} as const;
+
+const ColumnName = {
+  ID: 'id',
+  CHARACTER: 'character',
+  SETTING: 'setting',
+  SITUATION: 'situation',
+  PROP: 'prop',
+  TYPE: 'type',
+  GENRE_ID: 'genre_id',
+  CREATED_AT: 'created_at',
+  UPDATED_AT: 'updated_at',
+} as const;
+
+const PromptType = {
+  DAILY: 'daily',
+  MANUAL: 'manual',
+} as const;
+
+const CASCADE_RELATION_RULE = 'CASCADE';
+
+const up = (knex: Knex): Promise<void> => {
+  return knex.schema.createTable(TableName.PROMPTS, (table) => {
+    table.increments(ColumnName.ID).primary();
+    table.text(ColumnName.CHARACTER).nullable();
+    table.text(ColumnName.SETTING).nullable();
+    table.text(ColumnName.SITUATION).nullable();
+    table.text(ColumnName.PROP).nullable();
+    table
+      .enu(ColumnName.TYPE, [PromptType.DAILY, PromptType.MANUAL])
```

Comment:
That was valid for knex, but just to make it look more readabale
```suggestion
      .enum(ColumnName.TYPE, [PromptType.DAILY, PromptType.MANUAL])
```